server/leafnode.go

```diff
@@ -125,6 +125,58 @@ func validateLeafNode(o *Options) error {
 	if err := validateLeafNodeAuthOptions(o); err != nil {
 		return err
 	}
+	// In local config mode, check that leafnode configuration refers to accounts that exist.
+	if len(o.TrustedOperators) == 0 {
+		checkAccountExists := func(accName string, cfgType string) error {
+			found := false
+			if accName == "$G" || accName == _EMPTY_ {
```

Comment:
If you keep this here, and I am not convinced, honestly, there is no argument in favor of a `if/else` here:
```
if accName == globalAccountName || accName == _EMPTY_ {
    return nil
}
var found bool
for _, a := range o.Accounts {
...
```

Reply:
got simplified by use of a map